jscomp/others/belt_List.mli

```diff
@@ -271,7 +271,10 @@ val reverse: 'a t -> 'a t
 *)
 
 val mapReverse: 'a t -> ('a -> 'b) -> 'b t
-(** [mapReverse a f]
+[@@ocaml.deprecated "Use mapRight instead"]
+
+val mapRight: 'a t -> ('a -> 'b) -> 'b t
+(** [mapRight a f]
```

Comment:
the name `mapRight` does not feel right to me, what do other people think?

Reply:
Why do you need a map right?

Reply:
I agree... mapRight/mapReverse doesn't sound like something we should expose

Reply:
@cullophid for perf reasons, mapReverse is in general faster than map (more true in native backend), it is also available in original stdlib.

I feel `Reverse` is a better name than `Right` which does not assume the direction, note my original name is `Right`, but later changed to `Reverse` per feedback, now changed again per feedback...

Reply:
@bobzhang ah and because ocaml has sideeffects the order actually makes a difference.